dataprofiler/tests/profilers/test_datetime_column_profile.py

```diff
@@ -268,6 +268,8 @@ def test_profile(self):
             expected = defaultdict(float, {'datetime': 2.0})
             self.assertEqual(expected, profiler.profile['times'])
 
```

Comment:
nit pick but you can probably get rid of this extra space between the two `self.assert` in the same test function